codegen_glibmm/templates/proxy.h.templ

```diff
@@ -85,6 +86,10 @@ public:
     void {{ prop.name }}_set({{ prop.cpptype_in }}, const Gio::SlotAsyncReady &);
     void {{ prop.name }}_set_finish(const Glib::RefPtr<Gio::AsyncResult> &);
     {% endif %}
+    sigc::signal<void> &{{ prop.name }}_changed()
+    {
```

Comment:
move the `{` to the previous line

Reply:
Done.

(A bit confused about what the style is. { on same line for short "getters" in headers and { on new line in .cpp ?)

Reply:
Yes, in the same line when the function is inline, on the next one otherwise.